### ISSUE TITLE
your comment

### ISSUE THREAD

**Issue by aif2017** [jwyang/faster-rcnn.pytorch#534]
Hi  Man, i think you have done a very nice job in here, congrats. I would  just add the chance to finetune the network with other dataset, because  as it is, you assume people would want to finetune in the same COCO  Dataset. I made some minor modifications to make this happen:  Add a parameter with the number of classes (which could beinfered from the data/dataset.names) Freeze the darknet and then generate new output branches with the new right number of classes. Propagate the parameter through the anchor and mask creation  I will try to a PR with those changes so more people can benefit from it. Best regards!
--

pas chi shod MR?
این فایل را بفرسید

**Comment by jwyang** [comment]
great! thanks!
